muted-tests.yml

```diff
@@ -519,6 +519,8 @@ tests:
 - class: org.elasticsearch.xpack.esql.ccq.MultiClusterSpecIT
   method: test {csv-spec:fork.FiveFork}
   issue: https://github.com/elastic/elasticsearch/issues/134560
+- class: org.elasticsearch.upgrades.MatchOnlyTextRollingUpgradeIT
+  issue: https://github.com/elastic/elasticsearch/issues/134097
```

Comment:
Preemtively muting this test as it'll fail until the backport is completed. I've ran this test locally and it passes.

Reply:
even if this is temporary? My idea was to mute this test now, backport the fix, and then unmute the test. Without that, the test will fail against previous versions.